http/1.6/modules/ROOT/pages/http-request-configuration.adoc

```diff
@@ -0,0 +1,558 @@
+* URI Parameter value: `#[vars.customMap]`
+
+Before the request, assuming p1 already set: `#[vars.customMap ++ {'p2': 'customer'}]`
+
+HTTP Connector resolves parameters for each request, and evaluates DataWeave expressions in the context of the current message, in the order specified in the request. If a single parameter is defined more than once, the latest value is used.
+
+== Send Form Parameters
+
+To send form parameters in a `POST` request, follow these steps:
+
+. In Studio, select the HTTP *Request* operation from your flow.
+. In *General*, set *Method* to *POST*.
+. In *Body*, construct the payload of the Mule message as `application/x-www-form-urlencoded` with the names and the values of the parameters to send, for example:
+
+`[output application/x-www-form-urlencoded --- {'key1':'value1', 'key2':'value2'}`
+
+image::http-request-12.png[Body configuration with DataWeave expression]
+
+In the *Configuration XML* editor, the `body` configuration looks like this:
+
+[source,xml,linenums]
+----
+<flow name="httprequestFlow" >
+		<http:request method="POST" config-ref="HTTP_Request_configuration" path="testpath">
+			<http:body ><![CDATA[#[output application/x-www-form-urlencoded --- {'key1':'value1', 'key2':'value2'}]]]></http:body>
+		</http:request>
+	</flow>
+----
+
+The HTTP *Request* operation sends a POST request to the host location. The HTTP request contains `Content-Type: application/x-www-form-urlencoded` and the body is `key1=value1&key2=value2`.
+
+
+== Configure Round-Robin Requests
+
+The HTTP *Request* operation connects to configured hosts using round-robin DNS. Mule resolves all IP addresses associated with the specified host and performs load balancing by distributing the requests across all returned IPs.
+
+When the operation connects to resources that require authentication, the external service must replicate session information between IP addresses within the host of your service to avoid request rejection.
+
+If the external resource does not manage sticky sessions, add the service host name to the `mule.http.disableRoundRobin` system property. The HTTP request won't use round-robin DNS to connect to the configured host.
+
+Configure this system property when starting Mule by adding: +
+`./mule -M-Dmule.http.disableRoundRobin=serverhostname.com`
+
+== Configure Response Validator
+
+When the HTTP *Request* operation receives an HTTP response, the operation validates the response through the response status code. By default, HTTP Connector throws an HTTP error response when the status code is 400 or above. Consequently, if the server returns a 404 (Resource Not Found) or a 500 (Internal Server Error) a failure occurs and the connector triggers the error handling.
+
+Two ASCII `..` full stop characters define the range of failure status codes. Any value from 500 through 599 is considered a failure and raises an error. If the HTTP response has any other status code value, it's considered a success.
```

Comment:
It means that you can configure a range in a status code validator like this: "200..399"
In the XML, it would look like
```
<http:response-validator >
	<http:success-status-code-validator values="200..399" />
</http:response-validator>
```